app/views/content_items/detailed_guide.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, @content_item.title %>
+
+<%= render 'govuk_component/breadcrumbs', breadcrumbs: @content_item.breadcrumbs %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: "Guidance",
```

Comment:
Ah, that's why "Detailed Guidance" was renamed in "Detailed Guide" in the content schema example, probably to match with the Whitehall model name. I can amend the translation inside en.yml. Could it create problems somewhere else?